conf/messages.en

```diff
@@ -170,3 +174,14 @@ directorDetails.error.firstName.required = Enter the first name
 directorDetails.error.firstName.length = The first name cannot be more than 35 characters
 directorDetails.error.lastName.required = Enter the last name
 directorDetails.error.lastName.length = The last name cannot be more than 35 characters
+
+directorNino.title = Does the director have a National Insurance number?
+directorNino.heading = Does the director have a National Insurance number?
+directorNino.option1 = directorNino Option 1
+directorNino.option2 = directorNino Option 2
+directorNino.checkYourAnswersLabel = directorNino
```

Comment:
I think directorNino.option1 and directorNino.option2 should be removed and directorNino.checkYourAnswersLabel should be set